extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -389,7 +389,7 @@ public static function onParserAfterTidy( Parser &$parser, &$text ) {
 		return true;
 	}
 
-	public static function onShowEditLink( Parser &$this, &$showEditLink ) {
+	public static function onShowEditLink( Parser &$parser, &$showEditLink ) {
```

Comment:
maybe Kibana can tell us about production and if it threw any errors :)

on VOLDEV devbox it already throws a fatal because `$this` can't be used as parameter name